test/api/test_dataset_collections.py

```diff
@@ -113,11 +111,10 @@ def test_list_download(self):
 
     def test_pair_download(self):
         dataset_collection = self.dataset_collection_populator.create_pair_in_history(self.history_id).json()
+        self.dataset_collection_populator.wait_for_dataset_collection(dataset_collection, assert_ok=True)
         returned_dce = dataset_collection["elements"]
         assert len(returned_dce) == 2, dataset_collection
         returned_datasets = [dce["object"] for dce in returned_dce]
```

Comment:
test/api/test_dataset_collections.py:117:9: F841 local variable 'returned_datasets' is assigned to but never used

Reply:
Thanks @nsoranzo, I rebased with that change.